cluster/charts/crossplane-types/crds/core.oam.dev_applicationconfigurations.yaml

```diff
@@ -1,242 +0,0 @@
-
```

Comment:
Hrm, how are we going to get these OAM CRDs installed?

Reply:
hmm we could create a helm chart for `oam-kubernetes-runtime` and make it a dependency

Reply:
For posterity: we now run `go mod vendor` before generating our CRDs and point generation to the vendored `oam-kubernetes-runtime` directory :+1: